packages/xo-web/src/xo-app/hub/recipes/recipe-ev.js

```diff
@@ -0,0 +1,128 @@
+}
+
+const INDEX_TO_PERF_CONFIG = [
+  { cpu: 2, ram: 8 * 1024 * 1024 * 1024, diskSize: 10 * 1024 * 1024 * 1024 }, // 10GB
+  { cpu: 2, ram: 12 * 1024 * 1024 * 1024, diskSize: 250 * 1024 * 1024 * 1024 }, // 250GB
+  { cpu: 2, ram: 24 * 1024 * 1024 * 1024, diskSize: 550 * 1024 * 1024 * 1024 }, // 550GB
+  { cpu: 4, ram: 48 * 1024 * 1024 * 1024, diskSize: 750 * 1024 * 1024 * 1024 }, // 750GB
+  { cpu: 4, ram: 64 * 1024 * 1024 * 1024, diskSize: 1228 * 1024 * 1024 * 1024 }, // 1,2TB
+  { cpu: 8, ram: 96 * 1024 * 1024 * 1024, diskSize: 1536 * 1024 * 1024 * 1024 }, // 1,5TB
+  { cpu: 8, ram: 128 * 1024 * 1024 * 1024, diskSize: 2 * 1024 * 1024 * 1024 * 1024 }, // 2TB
+]
+
+export default decorate([
+  withRouter,
+  connectStore(() => ({
+    pools: createGetObjectsOfType('pool'),
+    recipeCreatingResources: state => state.recipeCreatingResources,
+  })),
+  provideState({
+    initialState: () => ({
+      selectedInstallPools: [],
+    }),
+    effects: {
+      async create() {
+        const recipeParams = await form({
+          defaultValue: {
+            pool: {},
+          },
+          render: props => {
+            return <RecipeForm {...props} value={{ ...props.value, timezone: DEFAULT_TIMEZONE }} />
+          },
+          header: (
+            <span>
+              <Icon icon='hub-recipe' /> {RECIPE_INFO.name}
+            </span>
+          ),
+          size: 'medium',
+        })
+
+        const {
+          vmName,
+          vmIpAddress,
+          gatewayIpAddress,
+          network,
+          sr,
+          sshKey,
+          timezone,
+          xoUsername,
+          xoPassword,
+          xoUrl,
+          dcScopeTemplateId,
+          userEmail,
+          userCompany,
+          performanceIndex,
+        } = recipeParams
+
+        const performanceConfig = INDEX_TO_PERF_CONFIG[performanceIndex]
+
+        const vmId = await createDcScopeVM({
+          vmName,
+          vmIpAddress,
+          gatewayIpAddress,
+          network: network.id,
+          sr: sr.id,
+          sshKey,
+          timezone,
+          xoUsername,
+          xoPassword,
+          xoUrl,
+          dcScopeTemplateId,
+          userEmail,
+          userCompany,
+          performanceConfig,
+        })
+
+        success(
+          _('recipeCreatedSuccessfully'),
+          <ButtonLink btnStyle='success' size='small' to={`/home?s=id:${vmId}&t=VM`}>
+            {_('recipeViewCreatedVms')}
+          </ButtonLink>,
+          8e3
+        )
+      },
+    },
+  }),
+  injectState,
+  ({ effects, recipeCreatingResources }) => (
+    <Card shadow>
+      <CardHeader>{RECIPE_INFO.name}</CardHeader>
+      <CardBlock>
+        <div
+          className='text-muted'
+          dangerouslySetInnerHTML={{
+            __html: marked(RECIPE_INFO.description),
+          }}
+        />
+        <hr />
+        <ActionButton block handler={effects.create} icon='deploy' pending={recipeCreatingResources[RECIPE_INFO.id]}>
```

Comment:
Why `block`? I don't think this prop is supported
```suggestion
        <ActionButton handler={effects.create} icon='deploy' pending={recipeCreatingResources[RECIPE_INFO.id]}>
```

Reply:
If I remove block, the button does not spread on the whole width of the component:
<img width="522" height="189" alt="image" src="https://github.com/user-attachments/assets/3dc47807-89e4-4fc1-a259-04b7bb5f9feb" />
<img width="519" height="184" alt="image" src="https://github.com/user-attachments/assets/64cb9c33-5f1f-4084-9b39-7d0862f7562e" />